src/main/java/com/gooddata/gdc/UriResponse.java

```diff
@@ -9,7 +9,9 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
- * Response containing URI
+ * Response containing URI string.
+ * <p>
+ * Intended for internal library purposes only, not to be used in public API!
```

Comment:
looking forward to java9 to avoid nonintentional exposing of internal structures ;-)